front/src/reducers/osrdconf/types.ts

```diff
@@ -29,6 +29,7 @@ export interface OsrdConfState extends InfraState {
   scenarioID?: number;
   pathfindingID?: number;
   timetableID?: number;
+  electricalProfileSetID?: number | null;
```

Comment:
Do you think it would be possible to have only a number or undefined ? You might need to add a test when you dispatch an action updateElectricalProfileSetId.
Also, I would like to get rid of the ID in maj, can you use Id in classic camel case ? 
```suggestion
  electricalProfileSetId?: number;
```

Reply:
Null has been removed so it's all good